connectors/cassandra-connector/src/main/scala/zio/connect/cassandra/CassandraConnector.scala

```diff
@@ -0,0 +1,25 @@
+package zio.connect.cassandra
+
+import zio.Trace
+import zio.connect.cassandra.CassandraConnector.{CassandraException, CreateKeySpaceObject}
+import zio.stream.ZSink
+
+trait CassandraConnector {
+
+  def createKeyspace(implicit trace: Trace): ZSink[Any, CassandraException, CreateKeySpaceObject, Boolean, Unit]
```

Comment:
The result type looks wrong. Instead of Boolean it should be the input type. Than slot is for leftovers.